docs/offboarding.rst

```diff
@@ -0,0 +1,162 @@
+Off-board Administrators and Journalists
+========================================
+
+When Journalists and SecureDrop Administrators leave your organization, it is important to off-board them from SecureDrop.
+
+**What you need:**
+
+- An *Admin Workstation*. Contact SecureDrop Support or follow our :doc:`guide to rebuilding an Admin Workstation <rebuild_admin>` if you do not have one.
+- An Admin account on the Journalist Interface
+
+.. important:: Note that, in particular for SecureDrop Administrators, additional measures may need to be considered if the user's departure is on unfavourable terms, and those measures will vary depending on the circumstances and your own internal incident response procedures. If you are in such a situation, consult the SecureDrop Support team or Digital Security Training team via the SecureDrop contact form, or email securedrop@freedom.press or a member of our team directly (found on the staff page). Note that, if you are in this situation, you may want to avoid reaching out to us via a ticket in our Support Portal, as that ticket will be visible to all Support Portal users at your organization.
+
+Off-boarding all users
+----------------------
+
+- `Inform the SecureDrop Support team`_ that the user's Support Portal account should be deactivated, and indicate if any new staff members should be added to the Support Portal..
+- De-activate the user's account on the Journalist Interface.
+- Retrieve *Admin Workstation* or *Journalist Workstation* USB drive(s), *Transfer*, *Export*, and *Backup* drive(s), and any other SecureDrop hardware or materials.
+- If the user receives email alerts (either OSSEC alerts or daily submission notifications), either directly or as a member of an email alias, remove them from those alerts and :ref:`set up someone new <ossec_guide>` to :ref:`receive those alerts <daily_journalist_alerts>`.
+- Make sure the user's organizational PGP key is revoked, if applicable, and proceed with other forms of off-boarding as appropriate to your organization.
+- (Circumstance-dependent) If you have specific concerns that the SecureDrop Submission Key has been compromised, you may want to :ref:`rotate the SecureDrop Submission Key <rotate_submission_key>`.
```

Comment:
The defined term is simply *Submission Key* ; since all of this is in a SecureDrop context, I think it's safe to remove the word "SecureDrop" in both usage instances.